src/components/attendee/ScheduleOverlay/helpers.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { MutableRefObject, RefObject } from "react";
```

Comment:
This is a code smell. If it is helpers, it should probably be a `.ts` file in `utils`, but if it has `.tsx` extension, there is probably a component hidden somewhere. Might be good to extract it into a component that uses one or two utility functions.